react/Header/UserAccount/UserAccount.less

```diff
@@ -127,22 +129,22 @@
   transition: transform 0.3s ease;
   transform-origin: 50% 50%;
   pointer-events: none;
-  .smallDeviceOnly({
+  @media @smallDeviceOnly {
     text-align: center;
     width: @grid-gutter-width;
     margin-right: (@grid-gutter-width / 2);
-  });
+  }
 }
 
 .chevronSvg {
   width: 14px;
   height: 14px;
   // Begin bold icon hacks
-  .smallDeviceOnly({
-    path {
+  @media @smallDeviceOnly {
+    path { // stylelint-disable-line selector-max-type
```

Comment:
You can avoid the comment with `:global(path)`, that's the intention of this rule in our codebase.